crates/forge-runner/src/running.rs

```diff
@@ -166,6 +160,55 @@ pub struct RunCompleted {
     pub(crate) fuzzer_args: Vec<String>,
 }
 
+pub struct VmExecutionContext<'a> {
+    pub runner: CairoRunner,
+    pub syscall_handler: SyscallHintProcessor<'a>,
+    pub initial_syscall_ptr: Relocatable,
+    // Additional data required for execution is appended after the program bytecode.
+    pub program_extra_data_length: usize,
+}
+
+fn initialize_execution_context<'a>(
```

Comment:
It could be a method of `VmExecutionContext`

Reply:
It's (very loosely at this point) based of blockifier code, I will add a relevant comment here.